build.gradle.kts

```diff
@@ -27,7 +27,7 @@ val profilerPlugins by configurations.creating
 dependencies {
     implementation("org.gradle:gradle-tooling-api:5.2.1")
     implementation("com.google.code.findbugs:annotations:3.0.1")
-    implementation("com.google.guava:guava:27.1-jre")
+    implementation("com.google.guava:guava:27.1-android")
```

Comment:
I think we should document the reason here.

Reply:
Done.